rllib/utils/replay_buffers/mixin_replay_buffer.py

```diff
@@ -0,0 +1,274 @@
+import collections
+import random
+from typing import Optional, Dict, Any
+
+from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID, SampleBatch, \
+    MultiAgentBatch
+from ray.rllib.utils.annotations import DeveloperAPI, override, ExperimentalAPI
+from ray.rllib.utils.replay_buffers.multi_agent_replay_buffer import \
+    MultiAgentReplayBuffer
+from ray.rllib.utils.typing import PolicyID, SampleBatchType
+from ray.rllib.execution.buffers.replay_buffer import _ALL_POLICIES
+
+
+@ExperimentalAPI
+@DeveloperAPI
```

Comment:
Nit: Let's just use `@ExperimentalAPI` for now. Delete the `@DeveloperAPI` decorator.